gui/src/components/onboarding/pages/body-proportions/ScaledProportions.tsx

```diff
@@ -7,11 +7,37 @@ import { HeightContextC, useProvideHeightContext } from '@/hooks/height';
 import { CheckFloorHeightStep } from './autobone-steps/CheckFloorHeight';
 import { ResetProportionsStep } from './scaled-steps/ResetProportions';
 import { DoneStep } from './scaled-steps/Done';
+import { useNavigate } from 'react-router-dom';
+import { useAppContext } from '@/hooks/app';
+import { useMemo } from 'react';
+import { MIN_HEIGHT } from './ProportionsChoose';
+import { ManualHeightStep } from './scaled-steps/ManualHeightStep';
 
 export function ScaledProportionsPage() {
   const { l10n } = useLocalization();
   const { applyProgress, state } = useOnboarding();
   const heightContext = useProvideHeightContext();
+  const navigate = useNavigate();
+  const { computedTrackers } = useAppContext();
+
+  const hmdTracker = useMemo(
+    () =>
+      computedTrackers.find(
+        (tracker) =>
+          tracker.tracker.trackerId?.trackerNum === 1 &&
+          tracker.tracker.trackerId.deviceId?.id === undefined
+      ),
+    [computedTrackers]
+  );
+
+  const beneathFloor = useMemo(
+    () =>
+      !(
+        hmdTracker?.tracker.position &&
+        hmdTracker.tracker.position.y >= MIN_HEIGHT
+      ),
+    [hmdTracker?.tracker.position?.y]
+  );
```

Comment:
We should probably only consider this if the headset is connected or not, the height could be very off, but could be correct with the floor height changed. Maybe we could give an option to use the manual height when the smol warning is shown?

Reply:
is this better?

Reply:
Looks good but any positional head tracker should work, though idk if it matters if we just go for HMD